src/charm.py

```diff
@@ -24,13 +23,16 @@
 from ops.main import main
 from ops.model import ActiveStatus, MaintenanceStatus
 
+from lib.charms.lte_core_interface.v0.lte_core_interface import (
```

Comment:
```suggestion
from charms.lte_core_interface.v0.lte_core_interface import (
```

Reply:
Actually, it's working when packed, but unit tests fail because are unable to find the lib?